test/integration/simple_zonal_private/testdata/TestSimpleZonalPrivate.json

```diff
@@ -83,7 +83,8 @@
         "displayName": "VPC"
       }
     ],
-    "enabled": true
+    "enabled": true,
+    "gcpPublicCidrsAccessEnabled": true
```

Comment:
Hi @laurentgrangeau - I think we're in the home stretch.  Let's add `gcp_public_cidrs_access_enabled = true` to a (non-private endpoin) cluster, and this asset to it's test data, and if that's green we should be good to merge!


```suggestion
```